src/types/types.tsx

```diff
@@ -1,6 +1,174 @@
+/**
+ * Custom types are defined here.
+ * They might need to be updated following the zaaktypebeheer-api.
+ */
+import { GridColDef } from '@mui/x-data-grid';
+import { Dispatch, SetStateAction } from 'react';
 import { NavigateFunction } from 'react-router-dom';
 import { AuthContextType } from '../components/Auth/Auth';
 
+//-----------------------//
+//    Data Structures    //
+//-----------------------//
+
+export type ZaaktypeT = {
+  url: string;
+  identificatie?: string;
+  omschrijving: string;
+  omschrijvingGeneriek?: string;
+  vertrouwelijkheidaanduiding:
+    | 'openbaar'
+    | 'beperkt_openbaar'
+    | 'intern'
+    | 'zaakvertrouwelijk'
+    | 'vertrouwelijk'
+    | 'confidentieel'
+    | 'geheim'
+    | 'zeer_geheim';
+  doel: string;
+  aanleiding: string;
+  toelichting?: string;
+  indicatieInternOfExtern: 'intern' | 'extern' | 'uitgaand';
+  handelingInitiator: string;
+  onderwerp: string;
+  handelingBehandelaar: string;
+  doorlooptijd: string;
+  servicenorm?: string | null;
+  opschortingEnAanhoudingMogelijk: boolean;
+  verlengingMogelijk: boolean;
+  verlengingstermijn?: string;
+  trefwoorden?: string[];
+  publicatieIndicatie: boolean;
+  publicatietekst?: string;
+  verantwoordingsrelatie?: string[];
+  productenOfDiensten: string[];
+  selectielijstProcestype?: string;
+  referentieproces: {
+    naam: string;
+    link?: string;
+  };
+  catalogus: string;
+  statustypen?: string[];
+  resultaattypen?: string[];
+
+  eigenschappen?: string[];
+  informatieobjecttypen?: string[];
+
+  roltypen?: string[];
+  besluittypen: string[];
+  deelzaaktypen?: string[];
+
+  gerelateerdeZaaktypen?: {
+    zaaktype: string;
+    aardRelatie: 'vervolg' | 'bijdrage' | 'onderwerp';
+    toelichting?: string;
+  }[];
+  beginGeldigheid: string;
+  eindeGeldigheid?: string;
+  versiedatum: string;
+  concept?: boolean;
+};
+
+export type InformatieObjectT = {
+  url?: string;
+  catalogus: string;
+  omschrijving: string;
+  zaaktype: string;
+  vertrouwelijkheidaanduiding: string;
+  beginGeldigheid: string;
+  eindeGeldigheid: string;
+  concept: boolean;
+};
+
+export type ZaaktypeInformatieRelatieT = {
+  url?: string;
+  zaaktype: string;
+  informatieobjecttype: string;
+  volgnummer: number;
+  richting: 'inkomend' | 'uitgaand' | 'definitief';
+  statustype?: string | null;
+};
+2;
+/**
+ * @deprecated must be replaced
+ * @use InformatieObjectenResposonse
+ */
+export type InformatieObjectenT = InformatieObjectenResposonse;
+/**
+ * @deprecated must be replaced
+ * @use ZaaktypenResponse
+ */
```

Comment:
What is the rationale behind this? Couldn't we just replace them now?

Reply:
✅ Should have been removed